scenes/Outline.shader

```diff
@@ -0,0 +1,32 @@
+/*
+ * Adapted from outline.shader obtained from GDQuest's godot-demos, 2020-04-01
```

Comment:
Add a better description for this shader.

Reply:
Improved description, describing in detail why there's a "multi outline" shader and how it differs from the regular outline shader.